layouts/shortcodes/section-hero.html

```diff
@@ -53,7 +61,10 @@ <h1 class="md-typescale-display-large">Call Detective</h1>
     left: 0;
     width: 100%;
     height: 100%;
+
     object-fit: cover;
+    image-rendering: pixelated;
```

Comment:
i think just omit this, `pixelated` is designed to preserve images with a pixelated style - think 8-bit art work, instead of using the normal smooth/bilinear filtering